### PR TITLE
update python 3.6 references to 3.9

### PR DESCRIPTION
**Pull request by snopoke**
Docs only changes so removing the PR template.

### CODE REVIEW

File: requirements/.python-version [hunk 1]
```diff
@@ -1 +1 @@
-3.6.9
+3.9.5
```
Comment (snopoke) [inline requirements/.python-version:1]: I'm not sure what the purpose of this file is

Comment (snopoke) [reply requirements/.python-version:1]: Ah, dependabot: https://github.com/dimagi/commcare-hq/commit/8aaa93d6dbef37c332dd1c8eb70d9f74c026ae86

File: .gitignore [hunk 2]
```diff
@@ -40,6 +40,7 @@ dump.rdb
 .transifexrc
 python_env
 python_env-3.6
+python_env-3.9
```
Comment (millerdev) [inline .gitignore:43]: Should we make this a wildcard so it's more future-proof?

```suggestion
python_env-*
```

**Review by millerdev — APPROVED** [review]
Thanks Simon

**Review by millerdev — APPROVED** [review]

File: DEV_SETUP.md [hunk 3]
```diff
@@ -198,7 +198,7 @@ Save those backups to somewhere you'll be able to access from the new environmen
     sudo pip install virtualenvwrapper --ignore-installed six
     ```
 
-- For downloading Python 3.6 consider:
+- For downloading Python 3.9 consider:
 
   - Using [pyenv](https://github.com/pyenv/pyenv-installer)
   - Using Homebrew with this [brew formula](https://gist.github.com/SamuelMarks/0ceaaf6d3de12b6408e3e67aae80ae3b)
```
Comment (Robert-Costello) [inline DEV_SETUP.md:204]: @snopoke  It looks like this formula is for installing Python 3.6.8. Does this need to be replaced also?

Comment (snopoke) [reply DEV_SETUP.md:204]: https://github.com/dimagi/commcare-hq/pull/30922

**Pull request merged**